packages/app/src/elements/audit.ts

```diff
@@ -54,8 +36,7 @@ export class Audit extends AuditMixin(StateMixin(Routing(View))) {
     ];
 
     render() {
-        const { _reusedPasswords, _weakPasswords, _compromisedPasswords } = this;
-
+        let items = app.auditedItems;
```

Comment:
There is no reason to re-run the audit every time we open this page since the information we want is already stored in the items themselves. This is more efficient and also guarantees that this view is always updated, even while open.